apigw-edge-custom-domain/README.md

```diff
@@ -0,0 +1,66 @@
+
+## Requirements
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+* Having your Route53 zone in the same AWS account.
+
+## Deployment Instructions
+1. Create the certificate in us-east-1
+    ```
+    $ sam deploy \ # Deploy command for SAM
+        -t certificate.yaml \ # the path to the SAM template to deploy the ACM certificate
+        --stack-name cert-sample-api \ # Specifies the name of the CloudFormation stack
+        --parameter-overrides "Domain=sample.com" \ # Overrides parameters defined in the SAM template
+        --region us-east-1 \ # Specifies the AWS region where the stack will be deployed, us-east-1 is mandatory for certificate
+        --resolve-s3 \ # Automatically create an Amazon S3 bucket to use for packaging and deploying
+        --capabilities CAPABILITY_IAM \ # Specifies the IAM capabilities required for the stack to create/modify IAM resources
+        --no-fail-on-empty-changeset \ # Indicates that the deployment should not fail if there are no changes to be made
+        --no-progressbar # Do not display a progress bar when uploading artifacts to Amazon S3
+    ```
+1. Create the main stack in your preferred region
+    ```
+    $ sam deploy \ # Deploy command for SAM
+        -t template.yaml \ # the path to the SAM template to deploy our application (EDGE API Gateway)
+        --stack-name sample-api \ # Specifies the name of the CloudFormation stack
+        --parameter-overrides "Domain=sample.com" \ # Overrides parameters defined in the SAM template
+        --region eu-west-1 \ # Specifies your preferred AWS region where the stack will be deployed, eu-west-1 for me
+        --resolve-s3 \ # Automatically create an Amazon S3 bucket to use for packaging and deploying
+        --capabilities CAPABILITY_IAM \ # Specifies the IAM capabilities required for the stack to create/modify IAM resources
+        --no-fail-on-empty-changeset \ # Indicates that the deployment should not fail if there are no changes to be made
+        --no-progressbar # Do not display a progress bar when uploading artifacts to Amazon S3
+    ```
+## How it works
+<img width="1273" alt="image" src="assets/architecture.png">
+
+In this pattern we use CloudFormation custom resources, you can check out this [workshop](https://mng.workshop.aws/cloudformation/customresource.html).
```

Comment:
Please include a small blurb like "resources which allow you to [...] For more information, please refer to this [...]"

Reply:
done